spec/features/user_visits_app_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.feature "User visits app" do
+
+  scenario "sees all the tasks" do
+    visit root_path
+
+    expect(page).to have_css('div.tasks', text: "Tasks")
```

Comment:
Nice, remind me to scope these assertions with CSS selectors when I add the styling!